import/popups/PasswordManagerDialog.qml

```diff
@@ -16,26 +16,61 @@ Dialog {
     id: dialog
 
     property alias notificationType: passwordManager.notificationType
+    property alias messageKey: passwordManager.messageKey
     property alias formData: passwordManager.formData
+    property alias selectedIndex: passwordManager.selectedIndex
     property alias _internalData: passwordManager._internalData
 
+    function _geckoKeyToString(key, formData) {
+        console.log("Converting localization key: " + key)
+        switch (key) {
+        case "updatePasswordMsg":
+            //% "Would you like to update password for user %1?"
+            return qsTrId("sailfish_components_webview_popups-la-update_password")
+                    .arg(formData["displayUser"])
+        case "updatePasswordMsgNoUser":
+            //% "Would you like to update the stored password?"
+            return qsTrId("sailfish_components_webview_popups-la-update_password_no_user")
+        case "rememberPasswordMsg":
+            //% "Would you like to save login and password for %1?"
+            return qsTrId("sailfish_components_webview_popups-la-save_password")
+                    .arg(formData["displayHost"])
+        case "rememberPasswordMsgNoUsername":
+            //% "Would you like to save password for %1?"
+            return qsTrId("sailfish_components_webview_popups-ls-save_password_no_user")
+                    .arg(formData["displayHost"])
+        case "passwordChangeTitle":
+            //% "Would you like to update the stored password"
+            return qsTrId("sailfish_components_webview_popups-ls-save_password_multiple_users")
+        case "userSelectText2":
+            //% "Login to update"
+            return qsTrId("sailfish_components_webview_popups-ls-save_password_select_login")
+        default:
+            return key
+        }
+    }
+
     PasswordManagerPopupInterface {
         id: passwordManager
 
         anchors.fill: parent
+        selectedIndex: selector.currentIndex
 
         Item {
             anchors.fill: parent
 
             DialogHeader {
+                id: header
                 //: Accept browser's request to save entered password
                 //% "Save"
-                acceptText: qsTrId("sailfish_components_webview_popups-he-accept_password_mgr_request")
+                acceptText:
```

Comment:
missing a qsTrId() here I think

Reply:
Thanks! I'm not sure why this wasn't generating an error, but I've put the string back in now at any rate.